CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 0.19.3 (2023-06-21)
+---------------------------
+Remove unreachable block
```

Comment:
There is no issue linked to this commit, is it intentional?

Reply:
yeah it is